group-coordinator/src/test/java/org/apache/kafka/coordinator/group/Assertions.java

```diff
@@ -198,6 +204,62 @@ private static void assertApiMessageAndVersionEquals(
                     }
                 }
             }
+        } else if (actual.message() instanceof GroupMetadataValue) {
```

Comment:
I understand that you replicated the pattern already in place. However, I don't like it even if I wrote it. Recently, I have been using a different approach which is better, I think.

I do the following:
1) I duplicate both messages (with `duplicate()` method)
2) I normalize them in place (e.g. sort lists, etc.)
3) I use `assertEquals` to compare them.

The benefit of this approach is that it automatically includes new fields. Would it work here too?

Reply:
Yeah it's more convenient. Let me change this.